frontend/src/metabase/visualizations/components/TableInteractive/cells/MiniBarCell.tsx

```diff
@@ -69,29 +67,119 @@ export const MiniBarCell = <TValue,>({
           borderBottomRightRadius: BORDER_RADIUS,
         };
 
+  if (isStatic) {
```

Comment:
It seems that static minibar is better to be its own component as they are way to different. I think it is cleaner to have it in `frontend/src/metabase/static-viz/components`